games-strategy/7kaa/7kaa-2.15.4p1.recipe

```diff
@@ -36,6 +38,7 @@ BUILD_REQUIRES="
 	devel:libcurl$secondaryArchSuffix
 	devel:libSDL2_2.0$secondaryArchSuffix
 	devel:libenet$secondaryArchSuffix
+	7kaa_music
```

Comment:
The music is a runtime requirement, not a build requirement, correct?

Reply:
I somehow manged to mess that up. I was rather tired yesterday